plugins/by-name/yuck/default.nix

```
@@ -0,0 +1,12 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.plugins.mkVimPlugin {
+  name = "yuck";
+  packPathName = "yuck.vim";
+  package = "yuck-vim";
+  globalPrefix = "yuck_";
```

Does this plugin needs `yuck` to be installed? If so, then:
```suggestion
  globalPrefix = "yuck_";
  
  dependencies = [ "yuck" ];
```
Also, add a commit before this one: `dependencies: add yuck`, adding `yuck.default = "yuck";` to `modules/dependencies.nix`.

---

Yes `vimPlugins.yuck-vim`

---

This is fine, it's already handled by the `package` option. I was talking about an extra dependency. But I think it's not the case, so just drop `dependencies = [ "yuck" ];`.